src/runtime/context.coffee

```diff
@@ -62,6 +63,22 @@ module.exports.Context = class Context
     else if @parent?
       @parent.getParentParameter name
 
+  getTimezoneOffset: ->
+    if @executionDateTime?
+      @executionDateTime.timezoneOffset
+    else if @parent?.getTimezoneOffset?
```

Comment:
The else if branch is not covered by the tests:
![image](https://user-images.githubusercontent.com/14879344/38055375-897bc3fa-32a7-11e8-8bf2-896705c7dd60.png)

Reply:
I think @jbradl11 added a test to cover this after @losborne made this comment.